src/test/domain/SqlDomain.test.ts

```diff
@@ -0,0 +1,91 @@
+import assert from 'assert';
+
+import { runtime } from '../runtime.js';
+
+describe('SqlDomain', () => {
+
+    context('psql', () => {
+        beforeEach(async () => {
+            const connectionUrl = runtime.testPostgresUrl;
+            await runtime.Sql.executeDefinition({ connectionUrl,
+                definition: 'CREATE TABLE test (id SERIAL PRIMARY KEY, username VARCHAR(50));' });
+        });
+        afterEach(async () => {
+            const connectionUrl = runtime.testPostgresUrl;
+            await runtime.Sql.executeDefinition({ connectionUrl,
+                definition: 'DROP TABLE test;' });
+        });
+
+        it('defines', async () => {
+            const connectionUrl = runtime.testPostgresUrl;
+            try {
+                const result = await runtime.Sql.executeDefinition({
+                    connectionUrl,
+                    definition: 'CREATE TABLE defined (id SERIAL PRIMARY KEY);'
+                });
+                assert.ok(result);
+            } catch (error: any) {
+                assert.fail(`Error thrown during table creation: ${error.message}`);
```

Comment:
Hm, I don't think I've seen a try-catch-rethrow pattern in any other tests; any particular reason to do it here?

Reply:
oops - no, I was fiddling with the return type of executeDefinition and this was overlooked. Thanks for spotting it!